src/index.ts

```diff
@@ -1,6 +1,15 @@
-import {fetchTarball, getPackument, namedScope, prepareManifest, publish, scopedOptions} from './util'
+import { fetchTarball, getPackument, namedScope, prepareManifest, publish, scopedOptions } from './util'
+import semver from 'semver';
```

Comment:
Codacy has a fix for the issue: [Replace `'semver'` with `"semver"`](https://app.codacy.com/gh/joebowbeer/regsync/pullRequest?prid=9263163)
```suggestion
import semver from "semver";
```